.gitignore

```diff
@@ -12,3 +11,4 @@ docs/
 npm-debug.log
 /.idea/
 .env
+dist/*
```

Comment:
We need `dist` into the repo to commit the built file on every release. This is why we had `build` ignored but not `dist`.

`build` is the build directory used in development and `dist` the one to store the built artifacts of each release.